LICENSE.txt

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2013 Joshua Holbrook
+Copyright (c) 2013-2015 Joshua Holbrook, Jon Ege Ronnenberg, James Halliday,
+and Google Inc.
```

Comment:
I'm making this PR for my work as a Google employee. It was requested that I add this to the copyright notice. As I personally understand things (I am not a lawyer), Google owns the copyright to the few lines being added. I added the largest contributors listed on GitHub to reflect their ownership as well.